src/index.html

```diff
@@ -198,7 +198,7 @@ <h2>Made with love.</h2>
           inspiring me to do this.</small></p>
     </div>
   </section>
-  <template>The boogeyman wrote all changes to the **master server** as he was all out of fucks to give. Thus, the slaves were read-only copies of master.</template>
+  <template>The boogeyman wrote all changes to the **master server** as he was running out of sympathy for these morons. Thus, the slaves were read-only copies of master.</template>
```

Comment:
I think this is pretty nice! Slightly profane. 🌶 I’ve also been using [`butt`](https://github.com/get-alex/alex/blob/b6283b2545332047348adbd98ceaa037e8a22a61/readme.md#checks) 🍑 as an example. Which do you prefer? I’m up for either!

Reply:
Oh, much better. Will return to this later and see what I come up with. Don't wanna be too flippant if it's going on the splash page. = P

Reply:
When I said later, I meant later that day, and didn't followup. Shame on me! Trying again for today.

Reply:
No problem, take your time!